_posts/2016-07-04-the-rush-age.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+name: The-Rush-Age
+title: The Rush Age
+date: 2016-07-04 10:00:00 +00:00
+author: Sergio Rodrigo Royo
+canonical:
+    name: my personal blog
+    href: http://srodrigo.me/?p=183
+image:
+    src: /assets/img/custom/blog/2016-07-04-the-rush-age/always-in-a-rush.jpg
+    attribution:
+      text: by Mayank Pandey
+      href: http://www.photo-street.com/2011/11/always-in-rush.html
+tags:
+- craftsmanship
+- practices
+alias: [/2016/07/04/The-Rush-Age]
+---
+
+We live in the rush age. We are literally overwhelmed by the amount of things we have to do, both at work and outside it. We leave work and have a bunch of emails, Facebook notifications, tweets, text messages (among others), waiting for response. Even worse, we are in bed keeping an eye open looking at our mobile phone in case we get a new notification. This is ridiculous.
+
+As software developers, the same thing happens at work. The feeling is that we need to be more and more productive. We have access to a **huge** amount of information and tools in order to achieve this (Google, Stack Overflow, that new library that claims to simplify your life 1% better than the one your are currently using, etc.). But the feeling I get is that all of this is making us consuming machines. We consume information, process it and pack it again as soon as possible. Our output? Mostly code. And how does this work? Let's see an example:
+
+So we are coding, and we get stuck. Here is a common algorithm:
+
+1. Look at the time tracking tool and see the task should already be finished; we are late. We feel the icy breath of the time on our nape.
```

Comment:
We feel the icy breath of the time on our nape. ==> We feel the icy breath of time on our nape.